source/install.txt

```diff
@@ -10,168 +10,242 @@ Install Compass
+         1. Open the {+download-page+}.
 
-         Once installed, |compass-short| launches automatically and prompts you to
-         configure privacy settings and specify update preferences.
+         #. Download the latest version of |compass| for Windows. The
+            |compass| installer is an ``.exe`` executable file.
 
      - id: ubuntu
        name: Ubuntu
        content: |
 
-         Download and Install from the MongoDB Downloads Page
-         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+         Download from the MongoDB Downloads Page
+         ````````````````````````````````````````
 
-         To install |compass| on Ubuntu, download the latest
-         |compass-short| package for Ubuntu from the {+download-page+}.
-         Double click on the ``.deb`` package to open the package using
-         Ubuntu Software. Click :guilabel:`Install`.
+         To download |compass| to an Ubuntu host using a web browser:
 
-         .. figure:: /images/compass/install-ubuntu.png
-            :figwidth: 654px
+         1. Open the {+download-page+}.
 
-         Once installed, launch Compass from your
-         :guilabel:`Applications` folder.
+         #. Download the latest version of |compass| for Ubuntu. The
+            |compass| installer is a ``.deb`` package.
 
-         Download and Install Using the Terminal
-         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+         Download using the Terminal
+         ```````````````````````````
 
-         You can also download and install |compass| via the terminal:
+         To download |compass| to an Ubuntu host using a shell:
 
-         #. Download the package:
+         1. Download the package using ``wget``:
 
             .. code-block:: sh
 
-               wget https://downloads.mongodb.com/compass/mongodb-compass_1.15.1_amd64.deb;
+               wget https://downloads.mongodb.com/compass/mongodb-compass_1.15.1_amd64.deb
 
-         #. Install Compass:
+            The |compass| installer is a ``.deb`` package.
 
-            .. code-block:: sh
+     - id: rhel
+       name: RHEL
+       content: |
 
-               sudo dpkg -i mongodb-compass_1.15.1_amd64.deb;
+         To download |compass| to a Red Hat Enterprise Linux host using
+         a shell:
 
-         #. Start Compass:
+         1. Download the package using ``wget``:
 
             .. code-block:: sh
 
-               mongodb-compass;
+               wget https://downloads.mongodb.com/compass/mongodb-compass-1.15.1.x86_64.rpm
 
-     - id: rhel
-       name: RHEL
+            The |compass| installer is an ``.rpm`` package.
+
+
+Install Compass
+---------------
+
+.. note::
+
+   As of MongoDB Server version 3.6, MongoDB packages a
+   platform-specific |compass-short| installation script with
+   `MongoDB Server downloads
+   <https://www.mongodb.com/download-center/enterprise>`_. For details,
+   see the :manual:`install_compass script
+   </reference/program/install_compass/>` documentation.
```

Comment:
Should we include a sentence stating the instructions on this page are if you are not using the installation script packaged with the server download?

Reply:
It's confusing (to me) that this script even exists, but I think your suggestion is fine @jeff-allen-mongo.